devp2p/rlpxcipher.py

```diff
@@ -405,11 +406,14 @@ def setup_cipher(self):
         else:
             self.egress_mac, self.ingress_mac = mac2, mac1
 
-        ciphername = 'aes-256-ctr'
-        iv = "\x00" * 16
+        # Yes, the encryption is insecure, see:
+        # https://github.com/ethereum/devp2p/issues/32
+        iv = b'\x00' * 16
         assert len(iv) == 16
-        self.aes_enc = pyelliptic.Cipher(self.aes_secret, iv, 1, ciphername=ciphername)
-        self.aes_dec = pyelliptic.Cipher(self.aes_secret, iv, 0, ciphername=ciphername)
+        cipher = Cipher(algorithms.AES(self.aes_secret), modes.CTR(iv), default_backend())
+        self.aes_enc = cipher.encryptor()
+        self.aes_dec = cipher.decryptor()
+
         self.mac_enc = AES.new(self.mac_secret, AES.MODE_ECB).encrypt
```

Comment:
Woudln't it be possible to use _Cipher(algorithms.AES(self.mac_secret),_ modes.ECB(), default_backend())_ here, to avoid depending on two different crypto libs?

Reply:
I was trying to keep my changes minimal.

Reply:
Also, changing this line won't let us get rid of pycryptodome dependency - we also use it for keccak.